apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -260,6 +260,13 @@ export const InfiniteEventTypeList = ({
     },
   });
 
+  const { data: team } = trpc.viewer.teams.get.useQuery({ teamId: group.teamId ?? -1 });
+
+  const isTeamAdminOrOwner =
+    team && (team.membership.role === MembershipRole.OWNER || team.membership.role === MembershipRole.ADMIN);
+
+  const isTeamPrivate = !!(team?.isPrivate && !isTeamAdminOrOwner);
```

Comment:
I don't think this is enough to protect the privacy. We're doing this stuff on the client. Which any member can inspect the HTML and check the NEXT_DATA in the json. Getting all users and emails. Which I don't think is desired. We should instead incorporate these changes in `trpc.viewer.eventTypes.getEventTypesFromGroup` query,

Reply:
Thanks for the explanation, completely agree with you. Made changes🙏🏽